### ISSUE TITLE
TypeError: LEVEL_PATTERN.exec(...) is null

### ISSUE THREAD

**Issue by blue-lightning** [Hafas/indiegala-auto-enter#12]
Error throw at line 241:
`minLevel: parseInt(LEVEL_PATTERN.exec(infoText)[1]),`

**Comment by Hafas** [comment]
Should've been fixed, yesterday. Have you updated to the current version (v.1.1.11) ?

**Comment by blue-lightning** [comment]
So changing the URL made the script unable to update. I should have noticed that.
Everything's fine now.

**Comment by Hafas** [comment]
What URL if I may ask?

**Comment by blue-lightning** [comment]
No, that's wrong; I was really sleepy last night when typing that. When looking at the newest commit you made to see what's up, I noticed the script header had no @downloadURL or @updateURL and assumed you've removed it. Turns out Greasemonkey checks update based on the script's install location and they are not needed anyway.

Now I don't know what stopped my Greasemonkey from force updating your script that night, but probably just a hiccup with the connection is all.
